### ISSUE TITLE
Crash report 

### ISSUE THREAD

**Issue by yogi-lo** [Tobi823/ffupdater#422]
Android pixel 6
No Google  account connected 

Stacktrace:
```
de.marmaro.krt.ffupdater.network.exceptions.NetworkException: File was not downloaded: /storage/emulated/0/Android/data/de.marmaro.krt.ffupdater/files/Download/org.chromium.chrome.zip
	at de.marmaro.krt.ffupdater.installer.ApkChecker$Companion.throwIfDownloadedFileHasDifferentSize(ApkChecker.kt:29)
	at de.marmaro.krt.ffupdater.BackgroundJob.downloadUpdateAndReturnAvailability(BackgroundJob.kt:248)
	at de.marmaro.krt.ffupdater.BackgroundJob.access$downloadUpdateAndReturnAvailability(BackgroundJob.kt:52)
	at de.marmaro.krt.ffupdater.BackgroundJob$downloadUpdateAndReturnAvailability$1.invokeSuspend(BackgroundJob.kt:0)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:749)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.7.14 (140) release |
| Device | Pixel 6 (oriole, oriole, oriole) |
| Manufacturer | google (Google) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 13 (SDK: 33) |
| OS | abfarm-release-2004-0228, android-build, release-keys, 1680732543000 |

**Comment by Tobi823** [comment]
Fixed in 78.0.1. If not, please create a new issue.

**Comment by yogi-lo** [comment]
Thank you.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
